hackathon_site/dashboard/frontend/src/testing/utils.tsx

```diff
@@ -131,6 +136,7 @@ export interface StoreEntities {
     pendingOrders?: OrderInTable[];
     teamDetailOrders?: OrderInTable[];
     teamDetailOrderState?: Partial<TeamOrderState>;
+    user?: Partial<UserState>;
```

Comment:
What is Partial?

Reply:
It's a partial state type because it means we don't need to input the full state (e.g. the error and loading properties) which gives us more flexibility when testing.

Reply:
makes sense!